app/presenters/vendor_api/note_presenter.rb

```diff
@@ -0,0 +1,19 @@
+module VendorAPI
+  class NotePresenter
+    attr_reader :note
+
+    def initialize(note)
+      @note = note
+    end
+
+    def as_json
+      {
+        id: note.id.to_s,
+        author: note.user.full_name,
+        message: note.message,
+        created_at: note.created_at.iso8601,
+        updated_at: note.updated_at.iso8601,
+      }
+    end
```

Comment:
We should follow the same pattern as the other presenters so that we can easily extend the note in a future version if required

```suggestion
    def to_json
       schema.to_json
    end
   
    def schema
      {
        id: note.id.to_s,
        author: note.user.full_name,
        message: note.message,
        created_at: note.created_at.iso8601,
        updated_at: note.updated_at.iso8601,
      }
    end
```